src/lib/braze.ts

```diff
@@ -0,0 +1,23 @@
+import { data as sd } from "sharify"
+
+export const setup = async () => {
+  if (
+    typeof window === "undefined" ||
+    !sd.BRAZE_API_KEY ||
+    !sd.BRAZE_API_URL ||
+    process.env.NODE_ENV !== "production"
```

Comment:
This might be silly but are both force staging and production setup with a NODE_ENV of "production"?

Reply:
Yeah. Generally `NODE_ENV` corresponds more to how the payload is being build/delivered. `development` would be used often for local builds that has a lot of extra instrumentation and stuff. `test` is used for unit test cases.